swiftui/ChartboostMediationDemo/Ads/AdTypeSelectionView.swift

```diff
@@ -46,27 +51,30 @@ struct AdTypeSelectionView: View {
             }
         }
     }
-}
 
-extension AdType {
-    @ViewBuilder var destination: some View {
-        GeometryReader { geometry in
-            ScrollView {
-                Group {
-                    switch self {
-                    case .banner:
-                        BannerAdView()
-                    case .interstitial:
-                        InterstitialAdView()
-                    case .rewarded:
-                        RewardedAdView()
+    func adView(forAdType adType: AdType) -> some View {
```

Comment:
nit: you could use @ViewBuilder to avoid having to explicitly declare a `adView` variable and return it. This makes the function work like other SwiftUI properties (e.g. body) and use the SwiftUI DSL instead of vanilla Swift

Reply:
Thanks, I have a lot to learn about SwiftUI and I appreciate the tip.